src/server/routes/entity/work.js

```diff
@@ -114,14 +114,28 @@ router.get(
 					.then((data) => entityToOption(data.toJSON()));
 		}
 
+		if (req.query.edition) {
+			propsPromise.edition =
+				Edition.forge({bbid: req.query.edition})
+					.fetch({withRelated: 'defaultAlias'})
+					.then((data) => entityToOption(data.toJSON()));
+		}
+
 		function render(props) {
 			if (props.creator) {
-				// add initial ralationship with relationshipTypeId = 8 (<Author> wrote <Work>)
+				// add initial ralationship with relationshipTypeId = 8 (<Work> is written by <Author>)
 				relationshipTypeId = 8;
 				initialRelationshipIndex = 0;
 				addInitialRelationship(props, relationshipTypeId, initialRelationshipIndex, props.creator);
 			}
 
+			if (props.edition) {
+				// add initial ralationship with relationshipTypeId = 10 (<Work> is contained in <Edition>)
+				relationshipTypeId = 10;
+				initialRelationshipIndex = 1;
```

Comment:
For here and everywhere else this mechanism is used, why not refactor to increment initialRelationshipIndex whenever it is used? That will prevent possible mistakes passing the wrong index manually :
```
let initialRelationshipIndex = 0
[...]
if (props.creator) {
addInitialRelationship(props, relationshipTypeId, initialRelationshipIndex++, props.creator);
}
if (props.edition) {
addInitialRelationship(props, relationshipTypeId, initialRelationshipIndex++, props.edition);
}
```

Sorry, I could have caught that earlier…